src/utils.rs

```diff
@@ -141,14 +142,90 @@ impl Console {
     }
 
     /// Read a chunk of raw data from the console and output it.
-    pub fn read_to(&self, output: &mut dyn Write) -> NitroCliResult<()> {
+    pub fn read_to(
+        &self,
+        output: &mut dyn Write,
+        disconnect_timeout_sec: Option<u64>,
+    ) -> NitroCliResult<()> {
+        fn epoll_cleanup(epoll_fd: i32, fd: i32) -> NitroCliResult<()> {
```

Comment:
no, please do not do this, it makes the code way harder to read. make it a  separate function if you really need to.
the right type for the parameters is RawFd.

Reply:
Could we please just use this: https://docs.rs/vmm-sys-util/0.8.0/vmm_sys_util/eventfd/struct.EventFd.html so that we  don't really care about cleanup.

Reply:
You are right with the parameter types. 
`epoll_cleanup` is only relevant in this context, why make a new function in this utils module, if it will only be used in this context?
We can't use EventFd because it's not only about closing the epoll socket, but more about making the epoll_ctl call with EpollCtlDel flag.